src/language/dafnyLanguageClient.ts

```diff
@@ -51,6 +52,21 @@ function getMarkGhostStatementsArgument(): string {
   return `--ghost:markStatements=${Configuration.get<string>(ConfigurationConstants.LanguageServer.MarkGhostStatements)}`;
 }
 
+function getDafnyPluginsArgument(): string {
+  let plugins = Configuration.get<string[]>(ConfigurationConstants.LanguageServer.DafnyPlugins);
+  if(plugins === null) {
+    return '';
+  }
+  plugins = plugins.filter(x => x !== null && x !== '');
+  const result = [];
+  for(const i in plugins) {
+    const possiblyEscaped = plugins[i].includes(' ') && !plugins[i].startsWith('"')
+      ? '"' + plugins[i].replace(/\\/g, '\\\\').replace(/"/g, '\\"') + '"' : plugins[i];
+    result.push(`--dafny:plugins:${i}=${possiblyEscaped}`);
```

Comment:
This is dangerous.  I have double-checked the escaping, but it gets injected right into another command line, which is scary.

Reply:
I removed this code, I now leave it entirely to the users to actually provide correct escape, which in most cases is not needed, and if needed, it matches the command-line escape that Dafny itself will unescape, not the plugin.